code/modules/antagonists/ert/ert.dm

```diff
@@ -189,3 +194,99 @@
 
 	missiondesc += "<BR><B>Your Mission</B> : [ert_team.mission.explanation_text]"
 	to_chat(owner,missiondesc)
+
+
+/datum/antagonist/ert/families
+	name = "Space Police Responder"
+	antag_hud_type = ANTAG_HUD_SPACECOP
+	antag_hud_name = "hud_spacecop"
+
+/datum/antagonist/ert/families/apply_innate_effects(mob/living/mob_override)
+	..()
+	var/mob/living/M = mob_override || owner.current
+	add_antag_hud(antag_hud_type, antag_hud_name, M)
+
+
+/datum/antagonist/ert/families/remove_innate_effects(mob/living/mob_override)
+	var/mob/living/M = mob_override || owner.current
+	remove_antag_hud(antag_hud_type, M)
+	..()
+
+/datum/antagonist/ert/families/greet()
+	to_chat(owner, "<B><font size=3 color=red>You are the [name].</font></B>")
+
+	var/missiondesc = "After an uptick in gang violence on [station_name()], you are responding to emergency calls from the station for immediate SSC Police assistance!\n"
+	missiondesc += "<BR><B>Your Mission</B>:"
+	missiondesc += "<BR> <B>1.</B> Secure the situation and crack down on any gang activity. You can view gangsters with your sunglasses."
+	missiondesc += "<BR> <B>2.</B> There is an undercover police officer on station. Secure him, receive his intel, and extract him safely."
+	missiondesc += "<BR> <B>3.</B> Minimize civilian casualties, but defend yourself and civilians from hostile gangsters."
```

Comment:
might want to say "non-gangster" instead of "civilian", just to be clear

Reply:
intentional, gangsters are still citizens of the SSC

Reply:
but their first objective literally tells them to "crack down" on gang activity

Reply:
Let's do it like the Combine do it, then. Violent gangsters are anticitizens.
"Attention citizens, miscount detected in your department. Please clear a path for SSC police units to allow detainment of the anticitizen."

Reply:
Lol @trollbreeder
@ATH1909 There's nothing stopping them from murdering gangsters rn as-is. This just gives them the opportunity to brutalize and not murder